src/Components/Table/Table.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import './styles.scss';
+
+const Table = ({ info }) => {
+  const columns = Object.keys(info[0]);
+  const tdData = () => {
+    return info.map((data, id) => {
+      return (
+        <tr key={id}>
+          {columns.map((v) => {
+            return <td key={v}>{data[v]}</td>;
+          })}
+        </tr>
+      );
+    });
+  };
+
+  return (
+    <table className="custom-table">
+      <tbody>{tdData()}</tbody>
+    </table>
+  );
```

Comment:
same as before, storing components in variables is not a good practice, using a function is not necessary, it is better to keep these in the return state to keep the file clean and understandable 

```jsx
return (
  <table className="custom-table">
    <tbody>
      {info.map((data, id) => (
        <tr key={id}>
          {columns.map(v => (
            <td key={v}>{data[v]}</td>
          ))}
        </tr>
      ))}
    </tbody>
  </table>
);
```